charts/dapr/README.md

```diff
@@ -92,10 +92,13 @@ The Helm chart has the follow configuration options that can be supplied:
 | `global.dnsSuffix`                        | Kuberentes DNS suffix                                                   | `.cluster.local`        |
 | `global.daprControlPlaneOs`               | Operating System for Dapr control plane                                 | `linux`                 |
 | `global.daprControlPlaneArch`             | CPU Architecture for Dapr control plane                                 | `amd64`                 |
-| `global.nodeSelector`                     | Pods will be scheduled onto a node node whose labels match the nodeSelector | `{}`                 |
-| `global.tolerations`                      | Pods will be allowed to schedule onto a node whose taints match the tolerations | `{}`                 |
-| `global.labels`                           | Custom pod labels                                                       | `{}`                 |
-| `global.k8sLabels`                        | Custom metadata labels                                                  | `{}`                 |
+| `global.nodeSelector`                     | Pods will be scheduled onto a node node whose labels match the nodeSelector | `{}`                |
+| `global.tolerations`                      | Pods will be allowed to schedule onto a node whose taints match the tolerations | `{}`            |
+| `global.labels`                           | Custom pod labels                                                       | `{}`                    |
+| `global.k8sLabels`                        | Custom metadata labels                                                  | `{}`                    |
+| `global.issuerFilenames.ca`               | Custom name of the file containing the root CA certificate inside the container  | `null`         |
```

Comment:
Although `null` is technically correct as it's the value passed to Dapr, I think you may be explicit here, such as `ca.crt` (even though that's set in Dapr and not in the charts)